internal/resolvers/youtube/helpers.go

```diff
@@ -17,3 +30,44 @@ func getYoutubeVideoIDFromURL(url *url.URL) string {
 func getYoutubeVideoIDFromURL2(url *url.URL) string {
 	return path.Base(url.Path)
 }
+
+func getChannelTypeFromString(channelType string) ChannelType  {
```

Comment:
Add a comment or a test for this function please

Reply:
Commented; not worth a test for something to trivial.